.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "freshli"]
+	banch = cli-breakout
```

Comment:
So I think this actually works (it pulls down the correct commit regardless).. but should be `branch`

Reply:
That is definitely an issue, thanks!